docs/getting_started/index.md

```diff
@@ -61,35 +50,23 @@ scenario_cfg = Config.configure_scenario("scenario", task_configs=[build_msg_tas
 ```
 
 - Lines 4-5 define the function that the task will use during execution.
-- Lines 8-9 configure the two data nodes, *input_name* and *message*.
+- Lines 8-9 configure the data nodes, *input_name* and *message*.
 - Line 10 configures a task called *build_msg* associated with the *build_message()*
   function, specifying the input and output data nodes.
 - Finally, line 11 configures the execution graph of the scenario providing 
   the previously configured task.
 
-### Core service run
-
-The Core service processes the configuration made in the previous step to set up the scenario 
-management feature.
-
-Include the following code in your Python script:
+2. **Core Service Initialization**: The Core service processes the configuration of the previous step to set up the scenario management feature.
```

Comment:
Well, it's not an initialization but a run.

Reply:
If it is not technical, don't you think it has a good english meaning and explains well what's happening?

Reply:
I prefer 'run' for 2 reasons: 
1. Even if we don't make it explicit, 'run' suggests that there is something 'alive'. A process, a service, or whatever. The term 'initialization' implies that once it is initialized, nothing happens, which is incorrect.
2. It is consistent with the method name.

Reply:
I don't really like it, to be honest. 'Core Service Run' needs some meaning to me.

For 2 reasons:

This is technically what is happening, but as the user sees it, it initializes the Core with the configuration loaded, and now he can create scenarios and tasks, ...

Run also has some other meaning in Data Science for pipeline runs for example.

I won't fight for it.